bin/configs/csharp-restsharp-name-mappings.yaml

```diff
@@ -10,6 +10,9 @@ parameterNameMappings:
   _type: UnderscoreType
   type_: TypeWithUnderscore
   http_debug_operation: HttpDebugOperation
+modelNameMappings:
+  Environment: Env
 additionalProperties:
   packageGuid: '{321C8C3F-0156-40C1-AE42-D59761FB9B6C}'
   hideGenerationTimestamp: "true"
+  targetFramework: net6.0
```

Comment:
Can you rename this file to include net6.0 to match the other files there? And do you really want it hard coded to 6.0? If you remove this line, then it uses the latest framework. In that case, you should put latest in the file name.

Reply:
> Can you rename this file to include net6.0 to match the other files there? 

done

> And do you really want it hard coded to 6.0? If you remove this line

yes. I target .net 6.0 for this sample because it's officially supported and the current setup in my machine only works with .net 6.0 (not 7.0) at the moment.